api/v1beta1/types.go

```diff
@@ -49,6 +49,9 @@ type Install struct {
 	DisableBootEntry bool `json:"disable-boot-entry,omitempty" yaml:"disable-boot-entry,omitempty"`
 	// +optional
 	ConfigDir string `json:"config-dir,omitempty" yaml:"config-dir,omitempty"`
+	// +optional
+	// +kubebuilder:default:="btrfs"
+	SnapshotterType string `json:"snapshotter-type,omitempty" yaml:"snapshotter-type,omitempty"`
```

Comment:
I wonder if it would be just enough naming it ` snapshotter` instead of  `snapshotter-type`. The later notation matches elemental-toolkit environment variables, but there is no explicit need to have a perfect match. Thoughts?

Reply:
What about `snapshotter.type`? 
I wonder if we'll need to expose other parameters, like most likely the `snapshotter.maxSnaps` eventually.

Reply:
Added `Snapshotter` struct in API. So this struct includes Type for now and opens the door to include further options in the future.